src/parser.h

```diff
@@ -434,6 +434,8 @@ class parser_t : public std::enable_shared_from_this<parser_t> {
     /// Remove the outermost block, asserting it's the given one.
     void pop_block(const block_t *expected);
 
+    /// Avoid maybe_t usage for ffi, sends a empty string in case of none.
+    wcstring get_function_name_ffi(int level);
```

Comment:
Is there a better way to handle `maybe_t` => Rust?

Reply:
The usual way is via unique_ptr. So this could return `unique_ptr<wcstring>` which would be NULL if the value were None.